docker-compose.yml

```diff
@@ -40,3 +40,53 @@ services:
       - ./code/bridge_data_v2:/home/robonet/code/bridge_data_v2
       - ./experiments:/home/robonet/experiments
       - /mount/harddrive/:/home/robonet/trainingdata
+
```

Comment:
Moving forward running the ML code outside of the robot code and docker container might be an easier way to go?

Reply:
Thanks for the feedback! Do we want to merge this pr for now or do we want to separate the ML and robot code before merging? Either way works for me.

Reply:
Sure we can merge this in since there are some cleanups that are helpful. Feel free to merge it in. 
Meanwhile, the orca eval code is in [this PR](https://github.com/rail-berkeley/orca/pull/29).